VideoLocker/src/androidTest/java/org/edx/mobile/test/SegmentTests.java

```diff
@@ -407,7 +407,7 @@ public void testScreenView() throws Exception {
     }
     public void testtrackUserDoesNotHaveAccount() throws Exception {
 
-        Properties props = segment.trackUserDoesNotHaveAccount();
+        Properties props = segment.trackUserSignUpForAccount();
```

Comment:
@hanningni - I had previously updated the method name in the following commit https://github.com/edx/edx-app-android/commit/2ab03e14507a487371c21e967e707456f0669aab which is merged in master in https://github.com/edx/edx-app-android/pull/91 but forgot to update it in <code>SegmentTest</code> class. Hence I have updated it.